openpype/hosts/max/api/lib.py

```diff
@@ -509,3 +509,18 @@ def get_plugins() -> list:
         plugin_info_list.append(plugin_info)
 
     return plugin_info_list
+
+
+@contextlib.contextmanager
+def suspended_refresh():
+    """Suspended refresh for scene and modify panel redraw.
+    """
+    rt.disableSceneRedraw()
+    rt.suspendEditing()
+
+    try:
+        yield
+
+    finally:
+        rt.enableSceneRedraw()
+        rt.resumeEditing()
```

Comment:
An issue in Maya implementation has been that similar logic failed in Maya batch / standalone. Should we check the same for Max whether this works fine or not in max standalone/batch mode?

If it doesn't work we might need to do some:

```python
if IS_HEADLESS:
    yield
    return
```

At the start of the context manager. I believe @antirotor did something recently in a maya PR.